crest/Assets/Crest/Crest/Scripts/Collision/CollProviderBakedFFT.cs

```diff
@@ -2,6 +2,8 @@
 
 // This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
 
+#if CREST_MATH
```

Comment:
I wonder if this should be more explicit, maybe CREST_UNITY_MATHEMATICS. Not a deal breaker for me though.

Reply:
Sure. Done for burst too to be consistent.